src/core_plugins/metrics/public/kbn_vis_types/editor_controller.js

```diff
@@ -1,6 +1,6 @@
 import modules from 'ui/modules';
 import '../services/executor';
-import 'plugins/timelion/directives/refresh_hack';
+import '../../../timelion/public/directives/refresh_hack';
```

Comment:
I created a #10971 to remove refresh-hacks all together based on my previous comment